styles/settings-view.less

```diff
@@ -570,13 +570,12 @@
       }
     }
 
-    .search-message,
-    .featured-message {
-      margin: @component-padding @component-padding 0 @component-padding;
+    .search-message {
```

Comment:
Tried to find if `.featured-message` gets used, but doesn't seem so. Probably a left-over. I think it's ok to remove it.